cmd/fs-v1.go

```diff
@@ -98,6 +99,22 @@ func newFSObjectLayer(fsPath string) (ObjectLayer, error) {
 	if fsPath == "" {
 		return nil, errInvalidArgument
 	}
+	fs, err := newFSObjects(fsPath)
+	fatalIf(err, "Unable to initialize FS object layer")
```

Comment:
We shouldn't fail here when called inside a function which returns error..

Reply:
ok, returning error instead.